ideascube/library/views.py

```diff
@@ -184,3 +184,34 @@ def get_row(self, book):
         return row
 
 book_export = staff_member_required(BookExport.as_view())
+
+
+class BookSpecimenExport(ZippedCSVExportMixin, View):
+
+    fields = ['item', 'barcode', 'serial', 'comments', 'location', 'file']
+    model = BookSpecimen
+    prefix = 'book_specimens'
+
+    def get_headers(self):
+        return self.fields
+
+    def get_row(self, bookspecimen):
+        row = {}
+
+        for field in self.fields:
+            if field == 'item':
+                row[field] = bookspecimen.item.name
```

Comment:
I was about to say "Yes!", but then I thought: what if we find the same ISBN but with a different title?
We already know different books share one ISBN, so this can happen. But more realistically if the title is somewhat different (case, missing "the", typo…) ?
I'd say:
- if we have an ISBN, just match on it, and we can't handle the case of different books sharing the ISBN, that will be a human responsibility (human creates the shit, human fixes it, no? :p)
- else match on the name, and if we have more than one, raise an error, human matter again